cvat/apps/engine/models.py

```diff
@@ -409,6 +409,7 @@ class ClientFile(models.Model):
     class Meta:
         default_permissions = ()
         unique_together = ("data", "file")
+        # note that Django will sort the files by filenames automatically
```

Comment:
Is this documented somewhere?

Reply:
I couldn't find neither the reason why it happens, nor mentions in documentation. It looks like it's just a detail of the implementation in Django.

Reply:
Then you shouldn't claim it in such definitive terms. You could note that Django may rearrange the files (although so far I don't see why it's relevant).